src/config.rkt

```diff
@@ -8,14 +8,16 @@
 ;; Flag Stuff
 
 (define all-flags
-  #hash([precision . (double)]
+  #hash([precision . (double fallback)]
+        [fn . (cbrt)]
```

Comment:
This is the right thing to do for now, but in the future we'll want to do something else for functions. In this case, I would add a `TODO` comment just to make clear that we don't intend for it to stay like this.